lib/ListActions/ListActions.js

```diff
@@ -485,6 +485,8 @@ var ListActions = module.exports = kind(
 		var listAction, i;
 
 		this.listActionComponents = [];
+		this.$.listActions.destroyClientControls();
```

Comment:
I didn't get a chance to fix this during the rush to tag, but we could guard this by checking `this.generated`, so that we do not run this unnecessarily at create time. We can include this in a future PR.